apps/web/app/future/org/[orgSlug]/embed/page.tsx

```diff
@@ -1,6 +1,4 @@
 import withEmbedSsrAppDir from "app/WithEmbedSSR";
-import type { PageProps as _PageProps } from "app/_types";
```

Comment:
Unused imports (@hbjORbj possibly a local lint issue?)

Reply:
yes i think, it's an linting issue.